library/bin/scaffold.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+const fs = require('fs-extra')
+const path = require('path')
+const spawn = require('cross-spawn')
+
+const appPath = process.cwd()
+const templatePath = path.resolve('../template')
+const useYarn = fs.existsSync(path.join(appPath, 'yarn.lock'))
+
+const appPackage = require(path.resolve(appPath, 'package.json'))
+
+appPackage.scripts = Object.assign({}, appPath.scripts, {
+  'build': 'NODE_ENV=production kaliber-build',
+  'lint': 'kaliber-lint',
+  'serve': 'kaliber-serve',
+  'serve:dev': 'PORT=8000 CONFIG_ENV=dev kaliber-serve',
+  'start': 'npm-run-all --parallel watch serve:dev',
+  'watch': 'CONFIG_ENV=dev kaliber-watch'
+})
+
+// write package.json
+fs.writeFileSync(
+  path.resolve(appPath, 'package.json'),
+  JSON.stringify(appPackage, null, 2)
+)
+
+// Copy the template
+fs.copySync(templatePath, appPath, { overwrite: false })
+
+// Rename gitignore after the fact to prevent npm from renaming it to .npmignore
+// See: https://github.com/npm/npm/issues/1862
+fs.move(
+  path.join(appPath, 'gitignore'),
+  path.join(appPath, '.gitignore'),
+  [],
+  err => {
+    if (err) {
+      // Append if there's already a `.gitignore` file there
```

Comment:
Wouldn't it be better to bail if the target directory is not empty?

Reply:
Short answer: no, long answer: the target directory cannot be empty since `kaliber-build` needs to be installed.